test/generate.test.js

```diff
@@ -0,0 +1,18 @@
+const { execSync } = require('child_process');
+const fs = require('fs');
+
+test('CLI generates correct SVG file', () => {
+
+  try {
+    const output = execSync(`node --experimental-specifier-resolution=node dist/index.js -p test/plan.json -o .`);
+
+    expect(output.toString()).toContain('SVG file saved successfully');
+
+    const outputSvgContent = fs.readFileSync('plan.svg', 'utf-8');
+    const expectedSvgContent = fs.readFileSync('test/expected.svg', 'utf-8');
+    expect(outputSvgContent).toEqual(expectedSvgContent);
+  
+} finally {
+    fs.unlinkSync('plan.svg');
```

Comment:
You might want to surround this with a try and an empty catch block.  Otherwise, if something fails in the generation of the plan file then the original error will be masked by the failure to delete plan.svg.

Reply:
added a catch block, thanks!